delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/eventlog/EventFetchRejection.scala

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.eventlog
+
+import io.circe.syntax._
+import io.circe.{Encoder, Json}
+
+final case class EventFetchRejection private (rejection: Json) extends Throwable(rejection.noSpaces)
```

Comment:
docs and extend from Exception instead, while suppressing the stacktrace

Reply:
should this be a ServiceError subtype instead (like in the case of ScopeInitialization)?